PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,21 @@
-This pull request `[DESCRIBE WHAT THIS PULL REQUEST IS TRYING TO ACCOMPLISH HERE]`. It `[DESCRIBE HOW IT ACCOMPLISHES THE GOAL HERE]`.
+Please add some information beneath all of these headers. Then, **delete this sentence**.
 
-See also: `[PROVIDE REFERENCES TO OTHER TICKETS, PULL REQUESTS, OR MAILING LIST THREADS HERE. OPTIONAL.]`
+### Brief summary of changes
+
+This Pull Request...
+
+### This resolves issue...
+
+- [ ] Redmine? #
+
+- [ ] Github? #
+
+### This PR improves LORIS by...
```

Comment:
This is redundant, if the summary isn't describing how it's improving LORIS, it's not a very good summary

Reply:
I see your point I had imagined the Summary as being more of a description of code changes whereas this section could be a more high-level contextual view.

ie.

```
## Summary

This PR prevents an error where the subclass was overriding a parent constructor when ti should not have been doing that. The functionality of its constructor was moved to another method. I also deleted some redundant code.

## This improves LORIS by...

Restoring the double data entry "View Details" links in the behavioural statistics module.

```

This definitely wasn't clear in my headers and I can change them. I feel that having a code-perspective and a context-perspective is valuable though. What are your thoughts?

Reply:
I agree with dave, there is a bit of overlap between the two, and I can see people either trying to copy-paste the contents of one into the other or adding filler text, especially when a small change is made. If there was only one section, it would allow for some flexibility, as people could choose to go into a bit of depth or not, depending on the changes made.

Reply:
I still don't think there should be two sections. The more sections there are to fill out, the less likely people are to do it. There should just be one good description.